frontend/src/pages/locations/locations.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Button } from "@mui/material";
+import { useNavigate } from "react-router";
+
+import { DummyLocation1 } from "mocks";
+import { Location } from "types";
+
+import { LocationCard } from "components/molecules";
+import Page from "../page";
+
+const LocationListPage = () => {
+  const navigate = useNavigate();
+
+  const [locations, setLocations] = React.useState<Location[]>();
+
+  React.useEffect(() => {
+    let isCancelled = false;
+
+    // TODO: call api and get events
+
+    !isCancelled &&
+    setLocations([
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      DummyLocation1,
+      ]);
+
+    return () => {
+      isCancelled = true;
+    };
+  }, []);
+
+  const onNewLocationClick = () => {
+    navigate("/locations/new");
+  };
+
+  const AddLocation = (
+    <Button variant="contained" color="primary" onClick={onNewLocationClick}>
+      New Infraestructure
+    </Button>
+  );
```

Comment:
we don't need this but can be removed later

Reply:
Got it! 👍